doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc

```diff
@@ -118,3 +129,4 @@ endif::PAM[]
 ifdef::DM[]
 Click *Finish* and then click *Deploy* to deploy the environment.
 endif::DM[]
+ 
```

Comment:
Not in this PR, but in the step "If you selected an external database server, provide the following information in additional fields:" what follows are sub-steps with bullets, but should be bullets with sub-bullets imo, since not sub-steps.